.github/workflows/main.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - CI
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  miniconda:
+    name: Miniconda ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+            os: ["ubuntu-latest"]
+    steps:
+      - name: check out diffpy.pdfmorph
+        uses: actions/checkout@v3
+        with:
+          repository: diffpy/diffpy.pdfmorph
```

Comment:
please make sure the read and try and understand what these are doing....you may guess that this won't work in the new repo after just copying it over....